Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: git://github.com/huboard/warden-github.git
-  revision: 459903681fa00820065348c27639b9f27db9fa55
+  revision: 3c3cf6943b01ff12cfe22a1ad1202ddcc4721e66
```

Comment:
^ updates our warden-github gem to the lastest git sha on https://github.com/huboard/warden-github